app/validators/blocks/calculation_block_validator.py

```diff
@@ -0,0 +1,52 @@
+from app.validators.blocks.block_validator import BlockValidator
+
+
+def key_value_is_unique(dictionaries: list[dict], key: str) -> bool:
```

Comment:
Might be clearer as `is_value_for_key_unique`?

Reply:
Updated 👍